runtime/lua/vim/_extui/shared.lua

```diff
@@ -77,7 +77,7 @@ function M.tab_check_wins()
     end
 
     if setopt then
-      api.nvim_buf_set_name(M.bufs[type], 'vim._extui.' .. type)
+      api.nvim_buf_set_name(M.bufs[type], 'extui//:vim._extui.' .. type)
```

Comment:
"nvim-pager" or "nvim-messages" would be a more appropriate scheme. 

idk why "extui" keeps popping up everywhere, it's not the name of this feature.

Reply:
It's just what I named the Lua module (because it is an implementation for the external UI events). It indeed shouldn't be referenced in any documentation or user facing stuff (like the buffer name) if/when this is promoted from experimental. Would be better to figure out what these buffers would be named sooner rather than later of course. `nvim-<type>` seems fine to me.